app/src/main/java/org/mozilla/focus/utils/AppConfigWrapper.java

```diff
@@ -101,6 +106,26 @@ public static boolean isLifeFeedEnabled(Context context) {
         return FirebaseHelper.getRcBoolean(context, FirebaseHelper.ENABLE_LIFE_FEED);
     }
 
+    public static String getLifeFeedProviderUrl(Context context, String provider) {
+        String source = FirebaseHelper.getRcString(context, FirebaseHelper.LIFE_FEED_PROVIDERS);
+        String url = "";
+
+        try {
+            JSONArray rows = new JSONArray(source);
+            for (int i = 0; i < rows.length(); i++) {
+                JSONObject row = rows.getJSONObject(i);
+                if (row.getString("name").equalsIgnoreCase(provider)) {
+                    url = row.getString("url");
+                    break;
+                }
+            }
+        } catch (JSONException e) {
+            e.printStackTrace();
```

Comment:
I think you should return the default value in this case

Reply:
Empty url may be better than broken(default) url